example/binary_request_execution/web-service/node-server.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const {
+  ClientServiceWithBinary,
+} = require('@scalar-labs/scalardl-node-client-sdk');
+const cors = require('cors');
+const app = express();
+const nodeServerPort = 3002;
+const bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+app.use(cors());
+app.listen(nodeServerPort,
+    () => console.log(`App listening at http://localhost:${nodeServerPort}`));
+const properties = {
+  'scalar.dl.client.server.host': '127.0.0.1',
+  'scalar.dl.client.server.port': 50051,
+  
```

Comment:
Sorry it's confusing to figure what is required and what is not but this property is required otherwise the request will fail
```suggestion
    'scalar.dl.client.server.privileged_port': 50052,
```